### PR TITLE
Match node list col grid sizes so heading and body columns align

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://jira.coreos.com/browse/CONSOLE-516

screen shot 2018-06-07 at 4 19 34 pm

### CODE REVIEW

**Comment by kans** [comment]
`isOperatorInstalled` is a property of each node:

```javascript
const isOperatorInstalled = node => _.has(node.metadata.labels, `${CLUO_PREFIX}id`);
```

It is possible that the flag exists for some nodes and not others.

**Comment by sg00dwin** [comment]
@spadgett  I updated the pr based on @kans comment.

File: frontend/public/components/node.jsx [hunk 1]
```diff
@@ -40,9 +40,9 @@ const Header = props => {
   const isOperatorInstalled = !!_.find(props.data, n => containerLinuxUpdateOperator.isOperatorInstalled(n));
   return <ListHeader>
     <ColHead {...props} className="col-xs-4" sortField="metadata.name">Node Name</ColHead>
-    <ColHead {...props} className={isOperatorInstalled ? 'col-xs-2' : 'col-xs-4'} sortFunc="nodeReadiness">Status</ColHead>
-    { isOperatorInstalled && <ColHead {...props} className="col-xs-3" sortFunc="nodeUpdateStatus">OS Update</ColHead> }
-    <ColHead {...props} className={isOperatorInstalled ? 'col-xs-3' : 'col-xs-4'} sortField="status.addresses">Node Addresses</ColHead>
+    <ColHead {...props} className="col-sm-2 col-xs-4" sortFunc="nodeReadiness">Status</ColHead>
+    <ColHead {...props} className="col-sm-3 hidden-xs" sortFunc="nodeUpdateStatus">OS Update</ColHead>
```
Comment (spadgett) [inline frontend/public/components/node.jsx]: Other tables consistently drop columns from right to left. Unless there's a good reason not to, I'd probably do that here as well.

Comment (sg00dwin) [reply frontend/public/components/node.jsx]: I was thinking the IP addresses might be more important, though if that's not the case I'll switch it.

**Comment by spadgett** [comment]
@sg00dwin linting fails with this error:

> 'isOperatorInstalled' is assigned a value but never used.

**Pull request merged**